src/node_worker.cc

```diff
@@ -109,10 +114,15 @@ Worker::Worker(Environment* env, Local<Object> wrap, const std::string& url)
     Isolate::Scope isolate_scope(isolate_);
     HandleScope handle_scope(isolate_);
 
-    isolate_data_.reset(CreateIsolateData(isolate_,
-                                          &loop_,
-                                          env->isolate_data()->platform(),
-                                          array_buffer_allocator_.get()));
+    IsolateData* isolate_data =
+        CreateIsolateData(isolate_,
+                          &loop_,
+                          env->isolate_data()->platform(),
+                          array_buffer_allocator_.get());
```

Comment:
I think you can keep this call as-is, and use `isolate_data_->set_options()` below?